.jenkins/sarus_run_test.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+set -x
+
+module load daint-gpu
+module add /project/d107/install/modulefiles/
+module load cray-python gcloud
+
+cd examples
+. /project/d107/install/venv/bin/activate 
+python create_arbitrary_rundir.py c48_config.yml "./c48_test"
+deactivate
+
+cd ${PWD}/c48_test
+cp ../../.jenkins/job_jenkins_sarus .
+
+export SCRATCH_DIR=${PWD}
+gcloud auth configure-docker
+
+dockerfiles=( $( ls ../../docker/Dockerfile.* ) )
+for df in ${dockerfiles[@]}; do
+   arch=${df:24}
+   export FV3_CONTAINER=fv3gfs-compiled:$arch
+   export TAR_FILE=${FV3_CONTAINER/":"/"_"}.tar
+   module load sarus
+   gsutil copy gs://vcm-ml-public/jenkins-tmp/${TAR_FILE}.gz .
+   gunzip ${TAR_FILE}.gz
+   sarus load ./${TAR_FILE} ${FV3_CONTAINER}
+   module unload sarus
+   sbatch --wait job_jenkins_sarus
```

Comment:
With this, I don't think we'll be able to see the slurm logs in jenkins... Do you know if these provide useful debugging information? You could use the buildenv, it has a launch_job command that will launch a slurm job, and poll it until it is done, and then you can cat the results. You can see an example in fv3util: https://github.com/VulcanClimateModeling/fv3gfs-python/blob/master/.jenkins/jenkins_slurm.sh, or fv3core (under development): https://github.com/VulcanClimateModeling/fv3core/blob/test-sarus/.jenkins/actions/run_regression_tests.sh